src/routes/plan/index.ts

```diff
@@ -9,6 +9,7 @@ router
   .use('/', planExercise)
   .get('/', AuthenticationMiddleware.authenticated, TrainingPlanController.getAll)
   .get('/:id', AuthenticationMiddleware.authenticated, TrainingPlanController.find)
+  .get('/creatorId/:creatorId', AuthenticationMiddleware.authenticated, TrainingPlanController.findByCreator)
```

Comment:
usually instead of creating a new endpoint for this, we create filter functionality on the getAll routes.

so we can fetch it like `/plans?creator=<creator-id>`

since we don't have much time lets go with this approach..

```suggestion
  .get('/creator/:creatorId', AuthenticationMiddleware.authenticated, TrainingPlanController.findByCreator)
```